hack/dev-setup

```diff
@@ -56,7 +68,7 @@ spec:
         name: etcd
         command:
         - etcd
-        - -advertise-client-urls=http://${MINIKUBE_IP}:2379
+        - -advertise-client-urls=http://${LOCAL_K8S_HOST_IP}:2379
```

Comment:
etcd does not start anymore because `${LOCAL_K8S_HOST_IP}` equals `localhost:6443` in the Docker case, hence, the start up parameter results in `- -advertise-client-urls=http://localhost:6443:2379`.

Reply:
should be fixed now. messed up the ip/port setup.

Reply:
also rebased my branch